android/KMAPro/build-play-store-notes.sh

```diff
@@ -1,55 +1,49 @@
-# Set sensible script defaults:
-# set -e: Terminate script if a command returns an error
-set -e
-# set -u: Terminate script if an unset variable is used
-set -u
+#!/usr/bin/env bash
 
-## START STANDARD BUILD SCRIPT INCLUDE
-# adjust relative paths as necessary
-THIS_SCRIPT="$(readlink -f "${BASH_SOURCE[0]}")"
-. "${THIS_SCRIPT%/*}/../../resources/build/build-utils.sh"
-## END STANDARD BUILD SCRIPT INCLUDE
+# Assumption: parent script that sources this already has START STANDARD BUILD SCRIPT INCLUDE
 
-QUIET=0
+# This script runs from its own folder
+cd "$THIS_SCRIPT_PATH"
```

Comment:
could I `cd $KEYMAN_ROOT/android/KMAPro`?

Reply:
It's a sourced script so it needs to preserve parent environment. Definitely don't change the folder here because that's a nasty side-effect (and could break anyway).

Could add `pushd "$KEYMAN_ROOT/android/KMAPro"` at start of `generateReleaseNotes` and `popd` at end of it.

Reply:
well, the calling script is already cd'ing there

Reply:
Sure, but we should be treating each script as isolated and 'black box' and not making assumptions about environment. So if we reuse that script elsewhere we don't end up with strange breakages in the future...